### PR TITLE
GET APIs for fetching Playlist and Track Info

### PR DESCRIPTION
**Pull request by isupreetk**
- Added seed data for playlist, playlist_details and playlist_tracks tables.
- Created routes and controllers for fetching all playlists for a particular logged in user & all tracks in given playlist ids.

### CODE REVIEW

**Comment by isupreetk** [comment]
POSTMAN collection Link -> https://brahm-bunga-dodra-kirtan-app.postman.co/workspace/Brahm-Bunga-Dodra---Kirtan-App-~7e67f0d1-2346-4de4-b314-3f63f24d045f/collection/28544015-21ba9cc0-b2d0-4c9d-9973-1590989cd235?action=share&creator=28544015

File: controllers/fetchKeyController.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export const fetchKey = (req: Request, res: Response) => {
       });
   } else {
     return knex("settings")
-      .where("key", searchParams.key)
+      .where("key", "in", [searchParams.key])
```
Comment (gdmmanmeet) [inline controllers/fetchKeyController.ts]: Can we merge both the queries into one "in" query? both are same at the moment

File: controllers/fetchPlaylistsController.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+export const fetchPlaylistsByUser = (req: Request, res: Response) => {
+  let { login_id } = req.query;
+
+  return knex("playlist")
+    .select("playlist_details.playlist_id", "playlist_details.playlist_name",
+            "playlist_details.playlist_code", "playlist_details.tracks_count",
+            "playlist_details.created_at", "playlist_details.created_by",
+            "playlist_details.modified_at", "playlist_details.modified_by")
+    .join("playlist_details", "playlist_details.playlist_id", "playlist.playlist_id")
+    .where("playlist.login_id", login_id)
```
Comment (gdmmanmeet) [inline controllers/fetchPlaylistsController.ts]: adding validation for login_id would be safe here

File: controllers/fetchTracksController.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        .then((data) => {
+            res.json(data);
+        })
+        .catch((error) => {
+            res.send(error);
+        });
+}
+else {
+    return knex("playlist_tracks")
+        .where("playlist_id", "in", [playlist_id])
```
Comment (gdmmanmeet) [inline controllers/fetchTracksController.ts]: combine both the queries

File: controllers/fetchKeyController.ts (hunk 1 above)
Comment (isupreetk) [reply controllers/fetchKeyController.ts]: Done.

File: controllers/fetchPlaylistsController.ts (hunk 2 above)
Comment (isupreetk) [reply controllers/fetchPlaylistsController.ts]: Done. Kindly review. Thanks.

File: controllers/fetchTracksController.ts (hunk 3 above)
Comment (isupreetk) [reply controllers/fetchTracksController.ts]: Done.

**Review by gdmmanmeet — APPROVED** [review]
LGTM

**Pull request merged**